### PR TITLE
Use routes for lists of resources

### LINKED ISSUE #7: Use routes for lists of assignments, pages, etc
`/assignments`, etc

Needed for #2

### PR DESCRIPTION
**Pull request by aaronshaf**
Resolves #7

### CODE REVIEW

**Comment by aaronshaf** [comment]
Example: http://5b91aadcfdd72a42662da562.common-cartridge-viewer-tmp.netlify.com/?src=https://dl2ge9eav9ucb.cloudfront.net/292b3b44b9b34309b7c6e1f92019007f.imscc#/

File: src/App.js [hunk 1]
```diff
@@ -116,17 +118,21 @@ export default class App extends Component {
...
 
         {this.state.cartridge != null && (
-          <CommonCartridge
-            onHistoryChange={this.handleHistoryChange}
-            src={this.state.cartridge}
-          />
+          <Router>
+            <CommonCartridge
+              onHistoryChange={this.handleHistoryChange}
+              src={this.state.cartridge}
```
Comment (jor3l) [inline src/App.js]: To keep things dry, can we just do a single Router and have the src as `this.state.cartridge || this.state.file`?

File: src/CommonCartridge.js [hunk 2]
```diff
@@ -746,118 +682,48 @@ export default class CommonCartridge extends Component {
                 padding="small"
                 background="default"
               >
-                <Router>
-                  <React.Fragment>
-                    <Switch>
-                      <Route
-                        exact
-                        path="/"
-                        render={() =>
```
Comment (jor3l) [inline src/CommonCartridge.js]: We should consider moving the renders to a component and have them in a views folder or something

Comment (jor3l) [reply src/CommonCartridge.js]: not a blocker tho, just a comment

File: src/App.js (hunk 1 above)
Comment (aaronshaf) [reply src/App.js]: That would assume either condition, but I don't think we can. We'll need to handle the absence of both.

File: src/CommonCartridge.js (hunk 2 above)
Comment (aaronshaf) [reply src/CommonCartridge.js]: Would definitely like to see CommonCartridge.js refactored -- it's getting big.

**Pull request merged**